### PR TITLE
Added Generated Code | Service Version 2020-10-02

### PR DESCRIPTION
**Pull request by mohsha-msft**

### CODE REVIEW

**Review by zezha-msft — CHANGES_REQUESTED** [review]

File: azblob/access_conditions.go [hunk 1]
```diff
@@ -8,22 +8,22 @@ import (
 type ModifiedAccessConditions struct {
 	IfModifiedSince   time.Time
 	IfUnmodifiedSince time.Time
-	IfMatch           ETag
```
[the remark this answers is not in the document]
Comment (mohsha-msft) [reply azblob/access_conditions.go:11]: Yes, the input of service has changed from *ETag to *String. We can keep it like before by defining ETag struct on our own.

[the remark this answers is not in the document]
Comment (zezha-msft) [reply azblob/access_conditions.go:11]: Can we avoid the breaking change with a README directive?

[the remark this answers is not in the document]
Comment (mohsha-msft) [reply azblob/access_conditions.go:11]: Sure. https://github.com/Azure/azure-storage-blob-go/pull/315 Here's another one.

**Comment by mohsha-msft** [comment]
https://github.com/Azure/azure-storage-blob-go/pull/315

**Pull request closed without merging**